pyexasol/http_transport.py

```diff
@@ -193,63 +345,19 @@ def __init__(self, connection, compression, table, import_params):
     def run_sql(self):
         table_ident = self.connection.format.default_format_ident(self.table)
 
-        parts = list()
-
-        if self.params.get("comment"):
-            comment = self.params.get("comment")
-            if "*/" in comment:
-                raise ValueError("Invalid comment, cannot contain */")
-            parts.append(f"/*{comment}*/")
-
-        parts.append(f"IMPORT INTO {table_ident}{self.build_columns_list()} FROM CSV")
-        parts.extend(self.build_file_list())
-
-        if self.params.get("encoding"):
-            parts.append(
-                f"ENCODING = {self.connection.format.quote(self.params['encoding'])}"
-            )
-
-        if self.params.get("null"):
-            parts.append(f"NULL = {self.connection.format.quote(self.params['null'])}")
-
-        if self.params.get("skip"):
-            parts.append(
-                f"SKIP = {self.connection.format.safe_decimal(self.params['skip'])}"
-            )
-
-        if self.params.get("trim"):
-            trim = str(self.params["trim"]).upper()
-
-            if trim != "TRIM" and trim != "LTRIM" and trim != "RTRIM":
-                raise ValueError("Invalid value for import parameter TRIM: " + trim)
-
-            parts.append(trim)
-
-        if self.params.get("row_separator"):
-            parts.append(
-                f"ROW SEPARATOR = {self.connection.format.quote(self.params['row_separator'])}"
-            )
-
-        if self.params.get("column_separator"):
-            parts.append(
-                f"COLUMN SEPARATOR = {self.connection.format.quote(self.params['column_separator'])}"
-            )
-
-        if self.params.get("column_delimiter"):
-            parts.append(
-                f"COLUMN DELIMITER = {self.connection.format.quote(self.params['column_delimiter'])}"
-            )
-
-        self.connection.execute("\n".join(parts))
+        import_query = ImportQuery.load_from_dict(
+            connection=self.connection, compression=self.compression, params=self.params
+        ).build_query(source=table_ident, exa_address_list=self.exa_address_list)
+        self.connection.execute(import_query)
 
 
 class ExaHttpThread(threading.Thread):
     """
-    HTTP communication and compression / decompression is offloaded to separate thread
-    Thread can be used instead of subprocess when compatibility is an issue
+    HTTP communication and compression / decompression is offloaded to a separate
+    thread. A thread can be used instead of a subprocess when compatibility is an issue.
```

Comment:
What is the intention of the last sentence?
Justify the design decision to use a thread? 
Is compatibility an issue, here? 
In which regard?

Reply:
This was already handled in https://github.com/exasol/pyexasol/pull/205, so i'll close it here.